src/org/scada_lts/utils/SystemSettingsUtils.java

```diff
@@ -762,4 +765,24 @@ public static int getValidationSearchCyclicDepth() {
             return defaultValue;
         }
     }
+
+    public static int getDefaultPurgePeriod() {
+        try {
+            String period = ScadaConfig.getInstance().getConf().getProperty(DEFAULT_PURGE_PERIOD, "1");
+            return Integer.parseInt(period);
+        } catch (Exception e) {
+            LOG.error(e.getMessage());
+            return 0;
+        }
+    }
```

Comment:
In the method above in this class of this method you have used the default value approach, you need to move it.